src/GraphQL/Execution/ExecutionError.cs

```diff
@@ -139,15 +139,15 @@ public ErrorLocation(int line, int column)
     public static class ExecutionErrorExtensions
     {
         /// <summary>
-        /// Adds a location to an <see cref="ExecutionError"/> based on a <see cref="AbstractNode"/> within a <see cref="Document"/>.
+        /// Adds a location to an <see cref="ExecutionError"/> based on a <see cref="ASTNode"/> within a <see cref="GraphQLDocument"/>.
         /// </summary>
-        public static TError AddLocation<TError>(this TError error, AbstractNode? abstractNode, Document? document)
+        public static TError AddLocation<TError>(this TError error, ASTNode? abstractNode, GraphQLDocument? document, string? originalQuery)
             where TError : ExecutionError
         {
-            if (abstractNode == null || document == null || document.OriginalQuery == null)
+            if (abstractNode == null || document == null || originalQuery == null)
```

Comment:
Should this be changed to also check if the source location was parsed?  The parser supports parsing without location information.

Reply:
👍 I'll check and be back with the answer.

Reply:
done